src/OpenConext/UserLifecycle/Domain/ValueObject/InactivityPeriod.php

```diff
@@ -19,28 +21,22 @@
 namespace OpenConext\UserLifecycle\Domain\ValueObject;
 
 use OpenConext\UserLifecycle\Domain\Exception\InvalidInactivityPeriodException;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+use Webmozart\Assert\Assert;
 
 class InactivityPeriod
 {
-    /**
-     * @var int
-     */
-    private $period;
 
-    public function __construct($inactivityPeriod)
-    {
-        if (!is_int($inactivityPeriod) || $inactivityPeriod <= 0) {
-            throw new InvalidInactivityPeriodException('The inactivity period must be an integer value');
-        }
+    public function __construct(
+        #[Autowire(param: 'inactivity_period')]
+        private readonly int $inactivityPeriodInMonths,
+    ) {
```

Comment:
This breaks with our usual services.yaml solution. You should be able to achieve the same goal with parameter binding?

Reply:
This is the new way of doing things as simple as possible

Reply:
As long as we do not introduce unwarranted backwards breaking changes I'm all for using new standards.